packages/tools/client-debugger/client-debugger/src/messaging/AudienceMessages.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IClient } from "@fluidframework/protocol-definitions";
+import { AudienceChangeLogEntry } from "../Logs";
+import { HasContainerId } from "./DebuggerMessages";
+
+import { IDebuggerMessage } from "./Messages";
+
+/**
+ * Metadata of clients within the Audience List
+ * @public
+ */
+export interface AudienceClientMetaData {
+	clientId: string;
+	client: IClient;
+}
+
+/**
+ * Inbound event requesting {@link AudienceChangeLogEntry} of the Container with the specific ID.
+ * Will result in the {@link AudienceEventMessage} message being posted.
+ *
+ * @public
+ */
+export interface GetAudienceMessage extends IDebuggerMessage<HasContainerId> {
+	type: "GET_AUDIENCE_EVENT";
+}
+
+/**
+ * Message data format used by {@link AudienceEventMessage}.
```

Comment:
Nit (we generally recommend leaving a blank line between the summary portion of a TSDoc comment and any other tags):
```suggestion
 * Message data format used by {@link AudienceEventMessage}.
 *
```

Reply:
Space added.